tutorials/image/cifar10/cifar10_multi_gpu_train.py

```diff
@@ -169,7 +176,7 @@ def train():
             # Calculate the loss for one tower of the CIFAR model. This function
             # constructs the entire CIFAR model but shares the variables across
             # all towers.
-            loss = tower_loss(scope)
+            loss = tower_loss(scope, images, labels)
```

Comment:
Doesn't this pass the same batch to every tower? I think we probably want to do `images, labels = cifar10.distorted_inputs()` within the `for i in xrange(FLAGS.num_gpus):` loop.